### PR TITLE
updated gettingstarted.rst

### PR DESCRIPTION
**Pull request by satendrakumar06**
Added new activator template.Which shows  how to write a Slick application using the cake pattern, writing unit test and providing a seed for starting with Slick.

### CODE REVIEW

**Comment by cvogt** [comment]
I think #1254 and #1255 should be one PR

**Comment by satendrakumar06** [comment]
@cvogt  You are right

File: slick/src/sphinx/gettingstarted.rst [hunk 1]
```diff
@@ -13,6 +13,9 @@ templates are created by the Slick team, with updated versions being made for ne
 
 * The `Slick Multi-DB Patterns template`_ shows you how to write Slick applications that can use
   different database systems and how to use custom database functions in Slick queries.
+  
+* The `Slick-starting-on-the-right-foot`_ shows you how write Slick applications using best practices. It also demonstrates unit testing and in general provides a seed for starting the product with Slick.
```
Comment (cvogt) [inline slick/src/sphinx/gettingstarted.rst]: I wouldn't call the cake pattern a best practice. It is one way to do things, but there are other ways with different trade-offs. Slick is not opinionated here. I would suggest to describe it more neutrally like "how to write a Slick application using the cake pattern".

**Comment by satendrakumar06** [comment]
@cvogt Is this ok  or create new combined pull request ?

File: slick/src/sphinx/gettingstarted.rst (hunk 1 above)
Comment (satendrakumar06) [inline slick/src/sphinx/gettingstarted.rst]: Thanks for suggestion.

**Comment by satendrakumar06** [comment]
@cvogt  Created new [pull request](https://github.com/slick/slick/pull/1256) with updated so now  closing this request

**Pull request closed without merging**